addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -285,12 +285,12 @@ def _l10n_eg_eta_prepare_invoice_lines_data(self, invoice, tax_data):
                     'amount': discount_amount,
                 },
                 'taxableItems': [
-                    {
-                        'taxType': tax['tax_id'].l10n_eg_eta_code.split('_')[0].upper().upper(),
-                        'amount': self._l10n_eg_edi_round(abs(tax['tax_amount'])),
-                        'subType': tax['tax_id'].l10n_eg_eta_code.split('_')[1].upper(),
-                        'rate': abs(tax['tax_id'].amount),
-                    }
+                    dict(
+                        taxType=tax['tax_id'].l10n_eg_eta_code.split('_')[0].upper().upper(),
+                        amount=self._l10n_eg_edi_round(abs(tax['tax_amount'])),
+                        subType=tax['tax_id'].l10n_eg_eta_code.split('_')[1].upper(),
+                        **{'rate': abs(tax['tax_id'].amount)} if tax['tax_id'].amount_type != 'fixed' else {}
+                    )
```

Comment:
Can't you just do this? Less diff, and same idea
```suggestion
                    {
                        'taxType': tax['tax_id'].l10n_eg_eta_code.split('_')[0].upper().upper(),
                        'amount': self._l10n_eg_edi_round(abs(tax['tax_amount'])),
                        'subType': tax['tax_id'].l10n_eg_eta_code.split('_')[1].upper(),
                        **({'rate': abs(tax['tax_id'].amount)} if tax['tax_id'].amount_type != 'fixed' else {}),
                    }
```

Reply:
Hi @gawa-odoo 

Thanks for the review.

Indeed, I should keep the diff as minimal as possible.
My bad!

I've updated the fix.